src/Get.tsx

```diff
@@ -200,10 +200,25 @@ class ContextlessGet<TData, TError> extends React.Component<
       this.setState(() => ({ error: null, loading: true }));
     }
 
-    const request = new Request(
-      url.resolve(base!, requestPath || path || ""),
-      this.getRequestOptions(thisRequestOptions),
-    );
+    /**
+     * If the path starts with slash, it is considered as absolute url.
+     * If not, it is considered as relative url.
+     * For example,
+     * base = "http://example.com/people" and path = "/absolute"
+     * reolves to "http://example.com/absolute"
+     * whereas,
+     * base = "http://example.com/people" and path = "relative"
+     * reolves to "http://example.com/people/relative"
+     */
+    const composedUrl = requestPath
```

Comment:
![rad](https://media.giphy.com/media/3o6gDPXMNxFhvHdcf6/giphy.gif)

This set of nested ternaries is so rad, but maybe we can refactor this a bit in order to preserve readability so that when we look at it in a few months we'll understand it faster.